### PR TITLE
Fix correct address on hba's postgresql file

### PR DESCRIPTION
**Pull request by ki0**
Like a puppetdb module that works with this.

### CODE REVIEW

File: manifests/postgresql.pp [hunk 1]
```diff
@@ -19,7 +19,7 @@
       postgresql::dbcreate { $foreman::db_name:
         role     => $foreman::db_user,
         password => $foreman::db_password,
-        address  => $foreman::db_server,
+        address  => '127.0.0.1/32'
```
Comment (alvagante) [inline manifests/postgresql.pp]: Please, place this value where $db_server's default is set: https://github.com/ki0/puppet-foreman/blob/master/manifests/params.pp#L117

**Comment by ki0** [comment]
Yes, you are right, but the tag in default statement, its wrong... mysql_grant?¿

**Comment by ki0** [comment]
Any problem about change the tag to potgresql_grants?¿

**Comment by alvagante** [comment]
Yeah, that's fine . Sorry for slow reaction

**Pull request merged**